packages/connector-sendgrid-mail/README.md

```diff
@@ -1,2 +1,86 @@
-### SendGrid Mail README
-placeholder
+# SendGrid mail connector
+
+SendGrid (also known as Twilio SendGrid) is a customer communication platform for transactional and marketing email. We can use its email sending function to send a "verification code" (or in other words, passcode).
+
+## Register and sign in SendGrid account
+
+Go to https://app.sendgrid.com/ to register a new account and sign in to SendGrid afterward. You may skip the registration step if you've already got an account.
+
+## Verify senders
+
+Senders are the email addresses where the emails containing verification codes come from. In order to send emails via the SendGrid mail server, you need to verify at least one sender.
+
+Starting from the [SendGrid console page](https://app.sendgrid.com/), go to _Settings_ -> _Sender Authentication_ from the sidebar.
+
+Domain Authentication is recommended but it is not obligated. You can click _Get Started_ in _Authenticate Your Domain_ card and follow the guide.
```

Comment:
```suggestion
Domain Authentication is recommended but not obligatory. You can click _Get Started_ in _Authenticate Your Domain_ card and follow the guide.
```

Reply:
<img width="1325" alt="image" src="https://user-images.githubusercontent.com/36393111/176345700-e7b4cbe7-63af-47ff-affa-58c2f438654f.png">

What's the difference if I click on the Create a sender at the home page?